ui/apps/search/src/components/routes/onboarding-page.tsx

```diff
@@ -52,23 +48,23 @@ const OnboardingPage: React.FC<OnboardingPageProps> = props => {
   };
 
   const toggleTagSubscribe = (tagName: string) => {
-    if (!isLoggedIn) {
+    if (!loggedProfileData) {
```

Comment:
You are only checking if user has logged in, you aren't using the profile data anywhere on the file :) `isLogged` from the `useLoggedIn` hook makes sense here. Perhaps you can fix this on your other PR where you refactored the hook.